basm/dto.go

```diff
@@ -15,6 +15,7 @@ func unmarshal(data []byte, v easyjson.Unmarshaler) error {
 	return easyjson.Unmarshal(data, v)
 }
 
+//easyjson:json
```

Comment:
If we tag all easyjson structs explicitly, then generating easyjson files via the makefile is the same for all files (rather than using the `easyjson --all` flag on some files and not others)